### ISSUE TITLE
.porter/ssl directory not created during install

### ISSUE THREAD

**Issue by rdarcy1** [konsulting/porter#29]
When running `site:secure` after new install:

```
In CertificateBuilder.php line 137:
                                                                                                                         
  file_put_contents(/Users/robin/.porter/ssl/entry-process.test.conf): failed to open stream: No such file or directory  
```

**Comment by rdarcy1** [comment]
```
In Filesystem.php line 122:
                                                                                                                            
  file_put_contents(/Users/robin/.porter/config/nginx/conf.d/entry-process.conf): failed to open stream: Permission denied  
```

**Comment by Keoghan** [comment]
@rdarcy1 I tested with a fresh install and didn't com across these - would you mind giving them a bash again at some point?

**Comment by rdarcy1** [comment]
Fixed in #54
